permissions/resource_permissions.go

```diff
@@ -215,6 +215,16 @@ func (a PermissionsAPI) Delete(objectID string) error {
 				accl.AccessControlList = append(accl.AccessControlList, change)
 			}
 		}
+
+		// handle special case when we add extra permission to a user home dir
+		if v, ok := d.GetOk("directory_path"); ok {
```

Comment:
We need to think how we will handle this for the case when we specify object ID instead of directory path...

Reply:
Yeah, that's a tricky one my brain refused to wrap itself around at 3am.. Directory ID won't match `user_name` of course, but fetching a user record based on a user id seemed like an overkill. Open for any hints here.

Reply:
yes, it's possible to get the directory path by object iD